blazor/rich-text-editor/toolbar.md

```diff
@@ -144,7 +146,49 @@ Quick toolbar is opened by clicking the table. It has different sets of commands
 {% endhighlight %}
 {% endtabs %}
 
-![Blazor RichTextEditor with Table Toolbar](./images/blazor-richtexteditor-quick-toolbar-table.png)
+![Blazor RichTextEditor table quick toolbar](./images/blazor-richtexteditor-quick-toolbar-table.png)
+
+### Audio quick Toolbar
+
+The rich text editor allows you to customize the inserted audio by using the [RichTextEditorQuickToolbarSettings.Audio](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.RichTextEditorQuickToolbarSettings.html#Syncfusion_Blazor_RichTextEditor_RichTextEditorQuickToolbarSettings_Audio) property. The audio quick toolbar has the following items.
+
+| Tools | Description |
+|----------------|--------------------------------------|
+| Replace | Can replace the audio with some other audio |
+| Remove| Delete the audio |
+| Display | Display the audio as inline or with break |
+
+{% tabs %}
+{% highlight razor %}
+
+{% include_relative code-snippet/custom-audio-quick-toolbar.razor %}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Blazor RichTextEditor audio quick toolbar](./images/blazor-richtexteditor-quick-toolbar-audio.png)
+
+### Video quick Toolbar
+
+The rich text editor allows you to customize the inserted video by using the [RichTextEditorQuickToolbarSettings.video](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.RichTextEditorQuickToolbarSettings.html#Syncfusion_Blazor_RichTextEditor_RichTextEditorQuickToolbarSettings_Video) property. The video quick toolbar has the following items.
+
+| Tools | Description |
+|----------------|--------------------------------------|
+| Replace | Can replace the video with some other video |
+| Align | Align the image with left, right and center |
+| Remove | Delete the video |
+| Layout option | Display the video as inline or with break |
+| Dimension | Can change an video dimensions, such as its height and width|
```

Comment:
Check the property name in source and then use in documentation

Reply:
Addressed